doc/CHANGELOG.md

```diff
@@ -16,10 +16,12 @@ With the portable version, you can easily store it in something like a USB disk,
 ### Fixed
 
 - Fix the wrong default setting for C++ executable file path.
+- Fix Font Size restores to small size after set to default or first launch.
```

Comment:
```suggestion
- Fix the displayed size of the default font is not the same as the actual size of the default font. (#425)
```

Reply:
Nope, this will change the font size to 9pt. Default font size is 12pt isn't it?